src/Parser/Cif.php

```diff
@@ -363,7 +363,9 @@ private static function parseXmlNode(\Gettext\Translations $translations, $filen
                 static::readXmlNodeAttribute($translations, $filenameRel, $node, 'custom-label', 'PageTypeComposerFormLayoutSetControlCustomLabel');
                 break;
             default:
-                throw new \Exception('Unknown tag name '.$path.' in '.$filenameRel."\n\nNode:\n".$node->ownerDocument->saveXML($node));
+                if (strpos($filenameRel, 'packages/') !== false) {
```

Comment:
Furthermore I'd `return` if we meet an unknown node in a package, otherwise the function parses the all the descendants of the node (and that's useless: since the parent node is unknown, also its children are unknown).